src/follower.py

```diff
@@ -7,6 +7,7 @@
 import picar
 
 from tagrec import TagRecognition
+from math import pi
```

Comment:
Remove this for two reasons:
1. You're not using it anywhere.
2. numpy has pi in it (np.pi)

Reply:
removed